README.md

```diff
@@ -313,3 +313,7 @@ This can by compiled using a command like the following after running
 `make install` on VkRunner:
 
     cc -o myrunner myrunner.c $(pkg-config --cflags --libs vkrunner)
+
+## Android
+
+See `android/README.md`
```

Comment:
I think you can make this a link:

     See [android/README.md](android/README.md)

?

Reply:
Done